src/Sulu/Bundle/AdminBundle/Resources/js/components/ArrowMenu/Item.js

```diff
@@ -25,31 +26,34 @@ export default class Item extends React.PureComponent<Props> {
 
     static defaultProps = {
         active: false,
+        disabled: false,
     };
 
     render() {
         const {
             children,
             active,
             icon,
+            disabled,
         } = this.props;
 
         const itemClass = classNames(
             itemStyles.item,
             {
                 [itemStyles.active]: active,
+                [itemStyles.disabled]: disabled,
             }
         );
 
         return (
-            <div className={itemClass} onClick={this.handleButtonClick}>
```

Comment:
I have just realized two things about this. First, it will break the webspace select (there won't be a single line for every webspace anymore) and that the `Action` component is already using a `button` instead of an `a` tag... Can you make a button out of this and style it accordingly, so that multiple webspaces in the dropdown are also shown correctly? And sorry about that, should have checked the other component first :see_no_evil: